pkg/build/info.proto

```diff
@@ -25,7 +25,7 @@ message Info {
   optional string go_version = 1 [(gogoproto.nullable) = false];
   optional string tag = 2 [(gogoproto.nullable) = false];
   optional string time = 3 [(gogoproto.nullable) = false];
-  optional string dependencies = 4 [(gogoproto.nullable) = false];
+  optional string revision = 4 [(gogoproto.nullable) = false];
```

Comment:
nit: we can probably get away with this here since I don't believe we ever pass a build.Info between nodes write it to disk, but reusing a field number makes me uneasy in general.